submissions/medinatapanpa/landingPage/index.html

```diff
@@ -0,0 +1,142 @@
+<!doctype html>
+<html>
+<head>
+    <title> Herbal</title>
+    <link rel="stylesheet" href="herbal.css">
+ </head>
+ <body>
+      <div class="mainbox">
+      <header>     
+            <div class="topnav">
+                <img class="logo"src="images/pistachios.jpg" alt="herbs">   
```

Comment:
the page preview does not show the images. Please rename `Images`folder to `images`

Reply:
done

Reply:
Not done

Reply:
PS C:\Users\Med's PC\Desktop\Dufuna-CodeCamp19\submissions\medinatapanpa\landingPage> git mv -f Images images
fatal: source directory is empty, source=submissions/medinatapanpa/landingPage/Images, destination=submissions/medinatapanpa/landingPage/images/Images 
message am getting.